src/index.js

```diff
@@ -57,3 +57,6 @@ export {default as ContinuousSizeLegend}
   from './lib/legends/continuous-size-legend';
 
 export {default as makeWidthFlexible} from './lib/make-vis-flexible';
+
+/* graph components */
+export * from './lib/graphs';
```

Comment:
Let's be explicit here on the exports from the graphs. This makes it clear which exports are from which file. Imagine when this file has more than one `export *` statement, it'd be difficult to find the source of an export without looking through all the `export *` files.